docs/reference/quickstart/run-elasticsearch-locally.asciidoc

```diff
@@ -0,0 +1,122 @@
+
+[discrete]
+[[local-dev-prerequisites]]
+=== Prerequisites
+
+If you don't have Docker installed, https://www.docker.com/products/docker-desktop[download and install Docker Desktop] for your operating system.
+
+[discrete]
+[[local-dev-env-vars]]
+=== Set environment variables
+
+Configure the following environment variables.
+
+[source,sh]
+----
+export ELASTIC_PASSWORD="<ES_PASSWORD>"  # password for "elastic" username
+export KIBANA_PASSWORD="<KIB_PASSWORD>"   # Used _internally_ by Kibana, must be at least 6 characters long
+----
+
+[discrete]
+[[local-dev-create-docker-network]]
+=== Create a Docker network
+
+To run both {es} and {kib}, you'll need to create a Docker network:
+
+[source,sh]
+----
+docker network create elastic-net
+----
+
+[discrete]
+[[local-dev-run-es]]
+=== Run {es}
+
+Start the {es} container with the following command:
+
+[source,sh,subs="attributes"]
+----
+docker run -p 127.0.0.1:9200:9200 -d --name elasticsearch --network elastic-net \
+  -e ELASTIC_PASSWORD=$ELASTIC_PASSWORD \
+  -e "discovery.type=single-node" \
+  -e "xpack.security.http.ssl.enabled=false" \
+  -e "xpack.license.self_generated.type=trial" \
+  {docker-image}
+----
+
+[discrete]
+[[local-dev-run-kib]]
+=== Run {kib} (optional)
+
+To run {kib}, you must first set the `kibana_system` password in the {es} container.
+
+[source,sh,subs="attributes"]
+----
+# configure the Kibana password in the ES container
+curl -u elastic:$ELASTIC_PASSWORD \
+  -X POST \
+  http://localhost:9200/_security/user/kibana_system/_password \
+  -d '{"password":"'"$KIBANA_PASSWORD"'"}' \
+  -H 'Content-Type: application/json'
+----
+// NOTCONSOLE
+
+Start the {kib} container with the following command:
+
+[source,sh,subs="attributes"]
+----
+docker run -p 127.0.0.1:5601:5601 -d --name kibana --network elastic-net \
+  -e ELASTICSEARCH_URL=http://elasticsearch:9200 \
+  -e ELASTICSEARCH_HOSTS=http://elasticsearch:9200 \
+  -e ELASTICSEARCH_USERNAME=kibana_system \
+  -e ELASTICSEARCH_PASSWORD=$KIBANA_PASSWORD \
+  -e "xpack.security.enabled=false" \
+  -e "xpack.license.self_generated.type=trial" \
+  {kib-docker-image}
+----
+
+[NOTE]
+====
+The service is started with a trial license. The trial license enables all features of Elasticsearch for a trial period of 30 days. After the trial period expires, the license is downgraded to a basic license, which is free forever. If you prefer to skip the trial and use the basic license, set the value of the `xpack.license.self_generated.type` variable to basic instead. For a detailed feature comparison between the different licenses, refer to our https://www.elastic.co/subscriptions[subscriptions page].
+====
+
+[WARNING]
+====
+*DO NOT USE THESE INSTRUCTIONS FOR PRODUCTION DEPLOYMENTS*
+
+The instructions on this page are for local development only. Do not use these instructions for production deployments, because they are not secure.
```

Comment:
How about we have another section at the bottom to show an example on how to connect? It would benefit just showing the url, username + password connect example so its super obvious whats the next step to use with the client.

Reply:
Done